PrivacyPolicy.md

```diff
@@ -0,0 +1,25 @@
+# Privacy Policy
+
+By using the app DialogMusicPlayer you are consenting to our policies regarding the collection, use and disclosure of personal information set out in this privacy policy.
+
+## Collection of Personal Information
+
+We do not collect, store, use or share any information, personal or otherwise.
```

Comment:
> from The Software.

from the user, right? is it okay to leave it as "We DO NOT collect, store, use or share any information, personal or otherwise."?